base/common/python/pki/util.py

```diff
@@ -180,22 +180,42 @@ def load_properties(filename, properties):
     with open(filename) as f:
 
         lines = f.read().splitlines()
+        name = None
+        multi_line = False
 
         for index, line in enumerate(lines):
 
-            line = line.strip()
+            if multi_line:
+                # append line to previous property
 
-            if not line or line.startswith('#'):
-                continue
+                value = properties[name]
+                value = value + line
 
-            parts = line.split('=', 1)
+            else:
+                # parse line for new property
+
+                line = line.lstrip()
+                if not line or line.startswith('#'):
+                    continue
+
+                parts = line.split('=', 1)
+                if len(parts) < 2:
+                    raise Exception('Missing delimiter in %s line %d' %
+                                    (filename, index + 1))
 
-            if len(parts) < 2:
-                raise Exception('Missing delimiter in %s line %d' %
-                                (filename, index + 1))
+                name = parts[0].rstrip()
+                value = parts[1].lstrip()
```

Comment:
I think we might run into issue when we have multiline ending with space
Example: 
````
key=value1 \ <space>
continued
````
Since line 215 strips right space, can we directly call `strip()`?

Reply:
Line continuation (i.e. the backslash) should always be at the end of the line. If you have a space after that, the backslash will be used to escape the character immediately following the backslash (probably the space itself). Right now the code doesn't process escape characters, but we might in the future.